specs/WorkerPostMessage.md

```diff
@@ -0,0 +1,682 @@
+Post message support for Dedicated and Service Workers
+===
+
+# Background
+Currently, if developers want to post a message to or from a worker, they must
+first post a message to the main thread and then post a message to the worker
+from the main thread. This leads to increased load on the main thread.
+
+The WebView2 team is introducing APIs that allow messages to be posted directly
+from the Host app to dedicated and service workers, and vice versa. This
+eliminates the need for the main thread to act as an intermediary, thereby
+improving app performance and responsiveness.
+
+# Description
+We propose the following APIs:
+
+**PostWebMessageAsJson**: This API gives ability to post a message to
+dedicated/service worker. The worker receives the message by subscribing to
+message event of `self.chrome.webview`.
+

```

Comment:
The CoreWebView2 overload of PostWebMessageAsJson that has params= (String, List<Object>) doesn't have a matching proposal here because the DOM objects in the List are only applicable on the main thread, is that right?